lib/publiccloud/ssh_interactive.pm

```diff
@@ -40,18 +40,9 @@ sub ssh_interactive_tunnel {
 
     set_var('SERIALDEV_',               $serialdev);
     set_var('_SSH_TUNNELS_INITIALIZED', 1);
-}
-
-sub ssh_interactive_join {
```

Comment:
Just leaving a method "interactive_leave" is not that explanatory. I would prefer to keep the balance of "join+leave".

Reply:
agree, but I would like to do the refactoring afterwards.
From this https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12236#discussion_r610430599 you can see that we have `start` which corresponding to `join` and `stop` which corresponds to `leave`.
For me it's a bit double, so refactoring it later will clean it up.

Reply:
if you agree I would keep it in that PR